package.json

```diff
@@ -137,6 +138,7 @@
     "scriptjs": "2.5.8",
     "sinon": "7.2.2",
     "smartcharts-beta": "0.4.15",
+    "tar": "^4.4.8",
```

Comment:
what is this for?

Reply:
that is the sub of sub dependency update, but, i think we just wait until that dependency update then we can update it. for now, ill remove it